apps/web/components/ui/form/CheckboxField.tsx

```diff
@@ -1,36 +1,50 @@
 import React, { forwardRef, InputHTMLAttributes } from "react";
 
+import InfoBadge from "@components/ui/InfoBadge";
+
 type Props = InputHTMLAttributes<HTMLInputElement> & {
-  label: React.ReactNode;
+  label?: React.ReactNode;
   description: string;
+  infomationIconText?: string;
 };
 
-const CheckboxField = forwardRef<HTMLInputElement, Props>(({ label, description, ...rest }, ref) => {
-  return (
-    <div className="block items-center sm:flex">
-      <div className="min-w-48 mb-4 sm:mb-0">
-        <label htmlFor={rest.id} className="flex text-sm font-medium text-neutral-700">
-          {label}
-        </label>
-      </div>
-      <div className="w-full">
-        <div className="relative flex items-start">
-          <div className="flex h-5 items-center">
-            <input
-              {...rest}
-              ref={ref}
-              type="checkbox"
-              className="text-primary-600 focus:ring-primary-500 h-4 w-4 rounded border-gray-300"
-            />
+const CheckboxField = forwardRef<HTMLInputElement, Props>(
```

Comment:
Update to the checkbox field 
-
As per recent designs checkboxes don't always have a "label" and are using the description as the label within the UI. Creates an easy an a11y way for this to be done in this component

Also added an Information Circle option - As this has also been present in a few designs